experiments/eval.py

```diff
@@ -0,0 +1,385 @@
+        ).astype(np.uint8)
+    return image_goal
+
+
+def request_goal_language(instruction, text_processor):
+    """
+    Request a new goal language from the user.
+    """
+    # ask for new instruction
+    if instruction is None:
+        ch = "y"
+    else:
+        ch = input("New instruction? [y/n]")
+    if ch == "y":
+        instruction = text_processor.encode(input("Instruction?"))
+    return instruction
+
+
+##############################################################################
+
+def main(_):
+    assert len(FLAGS.checkpoint_weights_path) == len(FLAGS.checkpoint_config_path)
+
+    # policies is a dict from run_name to get_action function
+    policies = {}
+    for checkpoint_weights_path, checkpoint_config_path in zip(
+        FLAGS.checkpoint_weights_path, FLAGS.checkpoint_config_path
+    ):
+        assert tf.io.gfile.exists(checkpoint_weights_path), checkpoint_weights_path
+        checkpoint_num = int(checkpoint_weights_path.split("_")[-1])
+        run_name = checkpoint_config_path.split("/")[-1]
+        policies[f"{run_name}-{checkpoint_num}"] = load_checkpoint(
+            checkpoint_weights_path, checkpoint_config_path
+        )
+
+    assert isinstance(FLAGS.initial_eep, list)
+    initial_eep = [float(e) for e in FLAGS.initial_eep]
+    start_state = np.concatenate([initial_eep, [0, 0, 0, 1]])
+
+    # set up environment
+    env_params = {
+        "fix_zangle": 0.1,
+        "move_duration": 0.2,
+        "adaptive_wait": True,
+        "move_to_rand_start_freq": 1,
+        "override_workspace_boundaries": WORKSPACE_BOUNDS,
+        "action_clipping": "xyz",
+        "catch_environment_except": False,
+        "start_state": list(start_state),
+        "return_full_image": False,
+        "camera_topics": CAMERA_TOPICS,
+    }
+    widowx_client = WidowXClient(FLAGS.ip, FLAGS.port)
+    widowx_client.init(env_params, image_size=FLAGS.im_size)
+
+    # load goals
+    if FLAGS.goal_type == "gc":
+        image_goal = None
+        if FLAGS.goal_image_path is not None:
+            image_goal = np.array(Image.open(FLAGS.goal_image_path))
+    elif FLAGS.goal_type == "lc":
+        instruction = None
+
+    # goal sampling loop
+    while True:
+        # ask for which policy to use
+        if len(policies) == 1:
+            policy_idx = 0
+        else:
+            print("policies:")
+            for i, name in enumerate(policies.keys()):
+                print(f"{i}) {name}")
+            policy_idx = int(input("select policy: "))
+
+        policy_name = list(policies.keys())[policy_idx]
+        get_action, obs_horizon, text_processors = policies[policy_name]
+
+        # show img for monitoring
+        if FLAGS.show_image:
+            obs = widowx_client.get_observation()
+            while obs is None:
+                print("Waiting for observations...")
+                obs = widowx_client.get_observation()
+                time.sleep(1)
+            cv2.imshow("img_view", obs["full_image"])
```

Comment:
Can you convert `obs["full_image"]` to BGR channel order here so that it's displayed correctly? (open-cv uses BGR channel order while our images are RGB)